src/common/object.ts

```diff
@@ -55,3 +55,7 @@ export const nameRegex = new RegExp(/^(.){1,60}$/);
 export const socialHandleRegex = new RegExp(/^@?([\w-]){1,39}$/i);
 export const handleRegex = new RegExp(/^@?[a-z0-9](\w){2,38}$/i);
 export const descriptionRegex = new RegExp(/^[\S\s]{1,250}$/);
+// Originated from: https://emailregex.com
+export const emailRegex = new RegExp(
+  /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+);
```

Comment:
The above regex supports things like `user/@example.com` or `user*@example.com`. While that is technically valid according to RFC I would prefer the zod one since it is more aligned with what you usually get in web applications. We don't have to support each nuance.

Reply:
I would find it weird to have above emails in the db and common email providers do not support it either way.
<img width="532" alt="image" src="https://github.com/user-attachments/assets/ef67c276-b313-4a5a-a9b7-93ee5b3b19e6">

Reply:
Moved it :)